index.bs

```diff
@@ -990,6 +990,174 @@ while (true) {
 }
 </pre>
 
+## Complete example ##  {#example-complete}
+
+*This section is non-normative.*
+
+This example illustrates use of the closed and ready promises, opening
+of uni-directional and bi-directional streams by either the client or
+the server, and sending and receiving datagrams.
+
+<pre class="example" highlight="js">
+// Adds an entry to the event log on the page, optionally applying a specified
+// CSS class.
+function addToEventLog(text, severity = 'info') {
+  let log = document.getElementById('event-log');
+  let mostRecentEntry = log.lastElementChild;
+  let entry = document.createElement('li');
+  entry.innerText = text;
+  entry.className = 'log-' + severity;
+  log.appendChild(entry);
+
+  // If the most recent entry in the log was visible, scroll the log to the
+  // newly added element.
+  if (mostRecentEntry != null &&
+      mostRecentEntry.getBoundingClientRect().top <
+          log.getBoundingClientRect().bottom) {
+    entry.scrollIntoView();
+  }
+}
+
+// "Connect" button handler.
+async function connect() {
+  let url = document.getElementById('url').value;
```

Comment:
Since `url` doesn't change, it can be const.

Reply:
Fixed.